app/views/users/show.html.erb

```diff
@@ -1,10 +1,18 @@
-<p style="color: green"><%= notice %></p>
-
-<%= render @user %>
-
-<div>
-  <%= link_to "Edit this user", edit_user_path(@user) %> |
-  <%= link_to "Back to users", users_path %>
-
-  <%= button_to "Destroy this user", @user, method: :delete %>
-</div>
+<div class="posts">
+    <img src="<%= @user.photo %>" />
+    <h1><%= @user.name %></h1>
+    <h2><%= @user.bio %></h2>
+    <p><%= @user.posts_counter %></p>
+
+    <%= @user.posts.each do |post| %>

```

Comment:
- [ ] The same thing here. Use `<% @user.posts.each do |post| %>` instead of` <%= @user.posts.each do |post| %>`

- [ ] KIndly add also the link to the new_post form here, Exemple :point_down: 

```rb
<%= link_to "New post", new_user_post_path(@user)%>

```